evap/staff/templates/staff_evaluation_textanswers_quick.html

```diff
@@ -76,10 +77,12 @@ <h5 class="modal-title" id="hotkeys-modal-title">Hotkeys</h5>
                             {% trans 'There are no text answers for this evaluation.' %}
                         </span>
                     {% endif %}
-                    {% if next_evaluation %}
-                        <span>
-                            {% blocktrans with name=next_evaluation.full_name count answers=next_evaluation.num_unreviewed_textanswers %}The next evaluation "{{ name }}" has got {{ answers }} unreviewed text answer.{% plural %}The next evaluation "{{ name }}" has got {{ answers }} unreviewed text answers.{% endblocktrans %}
-                        </span>
+                    {% if next_evaluations %}
+                        {% for next_evaluation in next_evaluations %}
+                            <span data-index="{{ forloop.counter0 }}" data-evaluation="{{ next_evaluation.pk }}">
```

Comment:
`[data-index]` is a bit too generic for my taste. Would it suffice to include `[data-evaluation]` and depending on the currently visible next evaluation to change the `[href]` of `[data-url=next-evaluation]`? Or include a `[data-next-evaluation-url]` here?

Reply:
I don't think this would be enough. We don't only need the url, but also the title. I will rename it to data-next-evaluation-index